src/OrchardCore.Modules/OrchardCore.Contents/Sitemaps/ContentTypesSitemapSourceBuilder.cs

```diff
@@ -162,7 +161,7 @@ private static void PopulateLastMod(ContentItem contentItem, XElement url)
             if (contentItem.ModifiedUtc.HasValue)
             {
                 var lastMod = new XElement(_namespace + "lastmod");
-                lastMod.Add(contentItem.ModifiedUtc.GetValueOrDefault().ToString("yyyy-MM-ddTHH:mm:sszzz"));
+                lastMod.Add(contentItem.ModifiedUtc.GetValueOrDefault().ToString("yyyy-MM-ddTHH:mm:sszzz", CultureInfo.InvariantCulture));
```

Comment:
Why it doesn't respect the current culture?

Reply:
I respects the current culture, that's the issue. Check the issue linked issue.